framework/src/components/logger/index.js

```diff
@@ -18,13 +18,13 @@ const { createLogger } = require('./logger');
 const { config: defaultConfig } = require('./defaults');
 const validator = require('../../controller/validator');
 
-function createLoggerComponent(options = {}) {
+function createLoggerComponent(name, options = {}) {
```

Comment:
Idea of such factory methods to accept single object was that, we can define its configuration as JSON files if required at any time. So why not we just consider `name` as part of options.

Reply:
I don't want name to go through the validation, that's the reason I kept it separate

Reply:
Discussion with Nazar: Add name to options and so, to schema object => validate for alphabetical and some chars like `-:`

Reply:
Additionally since we fixed the name to `lisk-framework`, so this option should not be name rather `moduleInfo`